ansible/README.md

```diff
@@ -282,20 +282,24 @@ the path to your OpenWhisk `ansible` directory contains spaces. To fix this, ple
 without spaces as there is no current fix available to this problem.
 
 #### Changing limits
-The system throttling limits can be changed by modifying the `group_vars` for your environment. For example,
-mac users will find the limits in this file [./environments/mac/group_vars/all](./environments/mac/group_vars/all):
+The system throttling limits's default value is configured in this file [./group_vars/all](./group_vars/all),
```

Comment:
The system throttling limits's default value is ->
The default system throttling limits are